json/src/main/kotlin/ktx/json/serializer.kt

```diff
@@ -0,0 +1,42 @@
+package ktx.json
+
+import com.badlogic.gdx.utils.Json
+import com.badlogic.gdx.utils.JsonValue
+
+/**
+ * Wrapping interface around [com.badlogic.gdx.utils.Json.Serializer].
+ * Provides null-safety for the methods.
+ */
+interface KtxSerializer<T> : Json.Serializer<T> {
+  override fun write(json: Json, obj: T, knownType: Class<*>?)
+
+  override fun read(json: Json, jsonData: JsonValue, type: Class<*>?): T
+}
+
+/**
+ * Wrapping interface around [com.badlogic.gdx.utils.Json.Serializer].
+ * Provides null-safety and convenient interface for serializer
+ * that is only able to [read]. The [write] method is no-op.
+ */
+interface KtxReadOnlySerializer<T> : Json.Serializer<T> {
+  override fun write(json: Json, obj: T, knownType: Class<*>?) = Unit
```

Comment:
I know this might be inconsistent with LibGDX approach, but I think it would be less error-prone to throw an exception when attempting to serialize an object with such serializer, instead of silently ignoring the object during serialization. Imagine you save a lot of JSONs and some values are simply omitted with no warning whatsoever.

Reply:
Just wanted to be consistent with LibGDX. But I agree, exception would be much more helpful. I'll change it.